.github/workflows/scan.yml

```
@@ -14,69 +14,35 @@ on:
     branches:
       - "main"
 
+permissions: {}
+
 jobs:
   check-run:
     name: Check PR run
     uses: bitwarden/gh-actions/.github/workflows/check-run.yml@main
+    permissions:
+      contents: read
 
   sast:
-    name: SAST scan
-    runs-on: ubuntu-22.04
+    name: Checkmarx
+    uses: bitwarden/gh-actions/.github/workflows/_checkmarx.yml@main
     needs: check-run
+    secrets:
+      CHECKMARX_TENANT: ${{ secrets.CHECKMARX_TENANT }}
+      CHECKMARX_CLIENT_ID: ${{ secrets.CHECKMARX_CLIENT_ID }}
+      CHECKMARX_SECRET: ${{ secrets.CHECKMARX_SECRET }}
     permissions:
       contents: read
       pull-requests: write
       security-events: write
```

Will need `id-token: write` to support Azure OIDC auth
```suggestion
      security-events: write
      id-token: write
```